mongoengine/fields.py

```diff
@@ -104,8 +105,24 @@ def __init__(self, regex=None, max_length=None, min_length=None, **kwargs):
         self.regex = re.compile(regex) if regex else None
         self.max_length = max_length
         self.min_length = min_length
+        self._kwargs = kwargs
```

Comment:
I don't think this line is useful

Reply:
This line is for another abandoned approach which I should clean it up, thanks.